.gersemirc

```diff
@@ -1,6 +1,6 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/BlankSpruce/gersemi/master/gersemi/configuration.schema.json
 
-definitions: [cmake]
+definitions: [cmake, cmake/stubs]
 indent: 4
 line_length: 80
```

Comment:
Should we make this 120?  It's not merged yet, but https://github.com/tenstorrent/tt-umd/pull/186 has 120 for C/C++.

Reply:
I have no opinion on this.. @afuller-TT @blozano-tt does it matter?

Reply:
I put 120 in `tt-metal` https://github.com/tenstorrent/tt-metal/blob/main/.gersemirc#L5

And @blozano-tt put 80 here just 'cause that's what the default is.  Go for it!

Reply:
changed to 120